test/views/ImportPostponedVatRequestedStatementsSpec.scala

```diff
@@ -39,32 +45,27 @@ class ImportPostponedVatRequestedStatementsSpec extends ViewTestHelper {
   }
 
   "PostponedVatViewModel" should {
-    "correctly handle historiesWithDisplayData" in new Setup {
-      val result: Seq[StatementDisplayData] = PostponedVatViewModel.historiesWithDisplayData(postponedVatViewModel.statementsForAllEoris)
+    "correctly handle statementsDisplayData" in new Setup {
+      val result: Seq[StatementDisplayData] = viewModel.statementDisplayData
```

Comment:
we already have viewModel.statementDisplayData saved as statementDisplayData in your trait

Reply:
This header belonged to the mapped .historiesWithDisplayData. This header has been moved now to .generateStatementDataItem and computed in there. We still display the same header in the view:
> @statement.dateHeader

Reply:
This has been addressed in commit [42fa837](https://github.com/hmrc/customs-historic-statement-frontend/pull/148/commits/42fa83718503df62412987860fce759b1b70f6ea)